pep-9999.txt

```diff
@@ -0,0 +1,200 @@
+PEP: 9999
+Title: Python local packages directory [We need to find a proper name]
+Version: $Revision$
+Last-Modified: $Date$
+Author: Kushal Das <mail@kushaldas.in>, Steve Dower <steve.dower@python.org>,
+        Donald Stufft <donald@stufft.io>, Nick Coghlan <ncoghlan@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 16-May-2018
+Python-Version: 3.8
+
+
+
+Abstract
+========
+
+This PEP proposes to add to Python a mechanism to automatically recognize a
+__pypackages__ directory and prefer importing packages installed in this
+location over user or global site-packages. This will share the standard library
+with the base installed Python. This will avoid the steps to create, activate
+or deactivate "virtual environments". Any Python scripts will use the
+``__pypackages__`` from the base directory of the script when present.
+
+
+
+Motivation
+==========
+
+Python virtual environments have become an essential part of development and
+teaching workflow in the community, but at the same time, they create a
+barrier to entry for many. The following are a few examples of the issues
+people get into while being introduced to Python (or programming for the first
+time).
+
+- How virtual environments work is a lot of information for anyone new. It takes
```

Comment:
Isn't this PEP just *even more* information for new people to assimilate?

Reply:
The idea/hope is that pip will start to install into `__pypackages__` by default some day, so it negates the need for beginners to learn about virtual environments and activating them or users screwing up their global install of Python.